src/openeo_grass_gis_driver/collections.py

```diff
@@ -89,5 +91,33 @@ def get(self):
                     #                       location, mapset)))
                     #     COLLECTIONS_LIST.append(ds)
 
+            # Additionally check for STAC collections registered in actinia
+            status_code, stac_collections = self.iface.get_stac_collections()
+            if status_code != 200:
+                # log.warning("Couldn't get STAC collections from actinia")
```

Comment:
Same here. Either this is a useful information or please remove commented code.

Reply:
Same here